spec/datasource.jest.ts

```diff
@@ -165,7 +165,7 @@ describe('GenericDatasource', () => {
     templateSrvStub.replace = (data) => data;
     setTemplateSrv(templateSrvStub);
 
-    ds.metricFindQuery('search', Format.Timeseries).then((result) => {
+    ds.metricFindQuery({query:'search', asJson: false}, Format.Timeseries).then((result) => {
```

Comment:
the type checker is off for spec, asjson is no longer there

Reply:
Whoops, thought I already did that in my previous commit but I think I missed it because these tests pass considering at part of the object isn't really considered in the test. Thanks for catching that.